components/Dialogs/src/MDCAlertController+Customize.h

```diff
@@ -14,14 +14,19 @@
 
 #import "MaterialDialogs.h"
 
+/**
+ The Customize extensions privde APIs that extend the default Material Dialog. Make sure to adjust
+ your custom theming when using these APIs, if you are concerned with conforming to the Material
```

Comment:
What does "Make sure to adjust your custom theming" mean in this context? If we're not able to explain in this context, can we link the reader to some additional documentation?